runtime/lua/vim/_defaults.lua

```diff
@@ -219,132 +219,150 @@ do
 
   --- vim-unimpaired style mappings. See: https://github.com/tpope/vim-unimpaired
   do
-    ---@param lhs string
-    ---@param rhs function
-    ---@param desc string
-    local function create_unimpaired_mapping(lhs, rhs, desc)
-      vim.keymap.set('n', lhs, function()
-        local _, err = pcall(rhs) ---@type any, string
-        if err then
-          vim.api.nvim_err_writeln(err)
-        end
-      end, { desc = desc })
+    --- Execute a command and print errors without a stacktrace.
+    --- @param opts table Arguments to |nvim_cmd()|
+    local function cmd(opts)
```

Comment:
This might be an example of the kind of thing to put in an internal-only `_util.lua` module, where useful stuff can live and potentially (but not always) graduate later to a public interface. (This particular thing might "graduate" to a flag on the existing `vim.cmd`.)